.github/workflows/python-tests.yml

```diff
@@ -11,10 +11,11 @@ on:
 jobs:
     build:
         runs-on: ubuntu-latest
+        name: "test py${{matrix.python-version}}"
         strategy:
             fail-fast: false
             matrix:
-                python-version: ["3.10"]
+                python-version: ["3.10", "3.12", "3.13"]
```

Comment:
Will this cause three sets of the build tests to run? If so that worries me since this queries a real car and can max out api calls.

Reply:
Thanks I didn't realize it could become a real problem. I left 3.12 since it's what matters at the moment and removed 3.10 (which was the previous value) and 3.13.